wd_helper.au3

```diff
@@ -632,7 +632,7 @@ Func _WD_FrameLeave($sSession)
 EndFunc   ;==>_WD_FrameLeave
 
 ; #FUNCTION# ===========================================================================================================
-; Name ..........: _WD_HighlightElement
+; Name ..........: [Deprecated] _WD_HighlightElement
```

Comment:
`[Deprecated]`
Why in `Name` and not in `Remarks` ?

Reply:
Because I wanted it to be highly visible. I also added remarks that attempt to address the situation.

I will have to check how this shows in the help file, and may have to make further adjustments.